nautobot/docs/user-guide/administration/tools/nautobot-server.md

```diff
@@ -58,7 +58,7 @@ If you have invalid filters in your `DynamicGroup` instances, the following outp
     DynamicGroup instance with name `Test DP 4` and content type `example_plugin | another example model` has an invalid filter `site`
 
 >>> Please fix the broken filters stated above according to the documentation available at:
-<nautobot-home>/static/docs/installation/upgrading-from-nautobot-v1.html#ui-graphql-and-rest-api-filter-changes
+<nautobot-home>/static/docs/user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.html#ui-graphql-and-rest-api-filter-changes
```

Comment:
This may need to be updated in code as well?

Reply:
@HanlinMiao can you look into this?